srcpkgs/musikcube/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'musikcube'
+pkgname=musikcube
+version=0.60.1
+revision=1
+build_style=cmake
+configure_args="-DNO_NCURSESW=1"
+makedepends="boost-devel libogg-devel libmicrohttpd-devel ffmpeg-devel \
+	libressl-devel pulseaudio-devel lame-devel zlib-devel alsa-lib-devel \
```

Comment:
no need for the `\` and use whitespace not tabs to indent newlines in the same variable